src/sql/base/browser/ui/table/table.ts

```diff
@@ -132,6 +132,24 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 
 		this._grid.onKeyDown.subscribe((e, args: Slick.OnKeyDownEventArgs<T>) => {
 			const evt = (e as JQuery.TriggeredEvent).originalEvent as KeyboardEvent;
+			if (evt.altKey && [37, 39].includes(evt.keyCode)) {
```

Comment:
as always, please add comment explaining the purpose and the future improvements.